spec.md

```diff
@@ -779,11 +779,9 @@ For success responses, the following fields are defined:
 }
 ```
 
-If the response contains `"state": "failed"` then the Platform MUST send a
-deprovision request to the Service Broker to prevent an orphan being created on
-the Service Broker. However, while the Platform will attempt
-to send a deprovision request, Service Brokers MAY automatically delete
-any resources associated with the failed provisioning request on their own.
+For asynchronous provision operations, if the response contains
+`"state": "failed"` then the Platform MAY need to perform
```

Comment:
Not sure the MAY here is appropriate because MAY is used to denote an optional thing that the Platform may or may not choose to do - it's up to it. However, in this case it's not up to the Platform, it's actually up to the orphan mitigation section. So I think saying something more like `Orphan Mitigation might be necessary, see the [Orphan Mitigation section](#orphan-migitiation) for more details.` might be better. Same for the cases below too.

Reply:
That makes complete sense. Thanks, changed!